tests/regression/16-relinv/README.md

```diff
@@ -0,0 +1,2 @@
+OSEK support has been removed from recent Goblint versions, please use Release 1.1.1
+Folder is left in place to avoid renumbering all tests
```

Comment:
Are you sure? Looks empty to me!

Reply:
Sorry, you're right. Apparently that was generated by the OSEK flag analysis and gitignored. So when I locally switched to this branch, the file remained because it was never tracked by git to begin with.